e2e/pipelines/test_standard_pipelines.py

```diff
@@ -165,3 +166,181 @@ def test_webqa_pipeline():
+    ]
+    document_store = InMemoryDocumentStore()
+    retriever = EmbeddingRetriever(
+        document_store=document_store, embedding_model="sentence-transformers/all-MiniLM-L6-v2"
+    )
+    document_store = InMemoryDocumentStore()
+    document_store.write_documents(documents)
+    document_store.update_embeddings(retriever)
+
+    docs_id: list = ["a", "b"]
+    filters = {"source": ["wiki3", "wiki4", "wiki5"]}
+    pipeline = MostSimilarDocumentsPipeline(document_store=document_store)
+    list_of_documents = pipeline.run_batch(document_ids=docs_id, filters=filters)
+
+    assert len(list_of_documents[0]) > 1
+    assert isinstance(list_of_documents, list)
+    assert len(list_of_documents) == len(docs_id)
+
+    for another_list in list_of_documents:
+        assert isinstance(another_list, list)
+        for document in another_list:
+            assert isinstance(document, Document)
+            assert isinstance(document.id, str)
+            assert isinstance(document.content, str)
+            assert document.meta["source"] in ["wiki3", "wiki4", "wiki5"]
+
+
+def test_summarization_pipeline():
+    docs = [
+        Document(
+            content="""
+    PG&E stated it scheduled the blackouts in response to forecasts for high winds amid dry conditions.
+    The aim is to reduce the risk of wildfires. Nearly 800 thousand customers were scheduled to be affected
+    by the shutoffs which were expected to last through at least midday tomorrow.
+    """
+        ),
+        Document(
+            content="""
+    The Eiffel Tower is 324 metres (1,063 ft) tall, about the same height as an 81-storey building, and the tallest
+    structure in Paris. Its base is square, measuring 125 metres (410 ft) on each side. During its construction,
+    the Eiffel Tower surpassed the Washington Monument to become the tallest man-made structure in the world, a
+    title it held for 41 years until the Chrysler Building in New York City was finished in 1930. It was the first
+    structure to reach a height of 300 metres. Due to the addition of a broadcasting aerial at the top of the tower
+    in 1957, it is now taller than the Chrysler Building by 5.2 metres (17 ft). Excluding transmitters, the Eiffel
+    Tower is the second tallest free-standing structure in France after the Millau Viaduct.
+    """
+        ),
+    ]
+    summarizer = TransformersSummarizer(model_name_or_path="sshleifer/distilbart-xsum-12-6", use_gpu=False)
+
+    ds = InMemoryDocumentStore(use_bm25=True)
+    retriever = BM25Retriever(document_store=ds)
+    ds.write_documents(docs)
+
+    query = "Eiffel Tower"
+    pipeline = SearchSummarizationPipeline(retriever=retriever, summarizer=summarizer, return_in_answer_format=True)
+    output = pipeline.run(query=query, params={"Retriever": {"top_k": 1}})
+    answers = output["answers"]
+    assert len(answers) == 1
+    assert "The Eiffel Tower is one of the world's tallest structures" == answers[0]["answer"].strip()
```

Comment:
Wait, this is very fragile, no?

Reply:
I have no clue, I've just copied the code... do you think we can improve it? I would imagine that if the random seeds are all fixed, this may be fairly robust actually